libs/utils/executor.py

```diff
@@ -551,8 +569,32 @@ def _wload_run(self, exp_idx, experiment):
             self._log.info('   %s',
                            stats_file.replace(self.te.res_dir, '<res_dir>'))
 
+        # Unfreeze the tasks we froze
+        if need_thaw:
+            self._thaw_userspace()
+
         self._print_footer()
 
+    def _freeze_userspace(self):
+        if 'cgroups' not in self.target.modules:
+            raise RuntimeError(
+                "freeze_userspace: devlib cgroups module not loaded. "
```

Comment:
This message is not completely clear on which module we are missing. What about:
```python
raise RuntimeError(
    "Userspace freezing failed. Ensure the 'cgroups' module is listed "
    "among your target/test devlib's modules")
```

Notice that I would also avoid the "freeze_userspace, we recently removed all these kind-of namespace prefixes.